Source/Cogworks.Essentials/Extensions/StringExtensions.cs

```diff
@@ -222,5 +224,51 @@ public static string RemoveNewLines(this string input)
                     .Replace("\r", string.Empty)
                     .Replace(Environment.NewLine, string.Empty)
                 : string.Empty;
+
+        public static string GetIpAddressWithoutPort(this string ipAddress)
```

Comment:
I'm not sure if these methods belong in the string extension

These seem to be very specific to IP addresses and URLs

Maybe a helper would be better?